src/main/java/de/varoplugin/cfw/version/ServerSoftware.java

```diff
@@ -30,7 +30,7 @@
 public enum ServerSoftware {
 
     MAGMA_1_18("Magma 1.18.2", SpigotVersionAdapter::new, "org.magmafoundation.magma.helpers.EnumJ17Helper", "Magma"),
-    MAGMA_1_12("Magma", versionSupplier -> new MagmaOneTwelveVersionAdapter(), "org.magmafoundation.magma.Magma", "Magma"),
+    MAGMA_1_12("Magma", null, "org.magmafoundation.magma.Magma", "Magma"),
```

Comment:
I think this should be `SpigotVersionAdapter::new`

Reply:
Ok, I just guessed this one